tests/unittests/distros/test_ubuntu.py

```diff
@@ -14,7 +17,7 @@ def test_package_command_only_refresh_snap_when_available(
             "cloudinit.distros.ubuntu.Snap.available",
             return_value=snap_available,
         )
-        m_snap_upgrade_packges = mocker.patch(
+        m_snap_upgrade_packages = mocker.patch(
```

Comment:
just noticed as spelling error on this variable in this test. Otherwise this test is unchanged.